client/components/common/DatePicker/DatePicker.jsx

```diff
@@ -0,0 +1,165 @@
+import React, {
+  useRef, useState, useEffect, useCallback,
+} from 'react';
+import ReactDayPicker from '@components/common/ReactDayPicker';
+import PropTypes from 'prop-types';
+import CalendarIcon from '@material-ui/icons/CalendarToday';
+import IconButton from '@material-ui/core/IconButton';
+import useOutsideClick from '@components/common/customHooks/useOutsideClick';
+import { makeStyles } from '@material-ui/core';
+
+// TODO: Apply gaps (margin, padding) from theme
+
+const useStyles = makeStyles(theme => ({
+  selector: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    width: '100%',
+    maxWidth: 268,
+    backgroundColor: theme.palette.primary.dark,
+    padding: 10,
+    borderRadius: 5,
+    fontSize: 14,
+  },
+  placeholder: {
+    color: theme.palette.text.secondaryDark,
+  },
+  selectorPopUp: {
+    position: 'fixed',
+  },
+  button: {
+    padding: 0,
+    color: theme.palette.text.dark,
+    '&:hover': {
+      backgroundColor: theme.palette.primary.dark,
+    },
+    '& svg': {
+      fontSize: 20,
+      fill: theme.palette.text.primary,
+    },
+  },
+}));
+
+const renderSelectedDays = (dates, classes, range) => {
+  const [from, to] = dates;
+  const isFromSelected = Boolean(from);
+  const isBothSelected = Boolean(from && to);
+
+  const selectedDaysElements = [];
+
+  if (isBothSelected) {
+    selectedDaysElements.push(
+      <span key="from">{from.toLocaleDateString('en-US')}</span>,
+      <span key="to">
+        {' '}
+        -
```

Comment:
add whitespace after the `-` to evenly space `from - to`
```suggestion
        -
        {' '}
```